reporter/src/test/java/org/arquillian/cube/q/recorder/TakeNetworkChaosInformationTest.java

```diff
@@ -95,12 +100,63 @@ public void should_capture_toxic_details_after_update() throws NoSuchFieldExcept
     }
 
     @Test
+    public void should_create_json_for_toxic_creation_and_write_it_to_file() throws IOException, NoSuchMethodException, NoSuchFieldException, IllegalAccessException, JSONException {
+        //given
+        ToxiProxyClient.Timeout timeout = new ToxiProxyClient.Timeout("timeout_downstream", "DOWNSTREAM", 1.0f, timeoutInMillis(1000));
+        ToxicCreated toxicCreated = new ToxicCreated(timeout);
+        ReporterConfiguration reporterConfiguration = new ReporterConfiguration();
+        Method method = getMethod("should_create_json_for_toxic_updation_and_write_to_file");
+        String path = getFilePath(reporterConfiguration, method.getName());
+
+        TakeNetworkChaosInformation takeNetworkChaosInformation = new TakeNetworkChaosInformation();
+        takeNetworkChaosInformation.propertyReportEvent = propertyReportEvent;
+        ArrayList<Map<String, Object>> toxics = getToxicList(takeNetworkChaosInformation);
+
+        //when
+        takeNetworkChaosInformation.captureToxicDetailsAfterCreate(toxicCreated, toxicAction);
+        takeNetworkChaosInformation.reportToxicConfiguration(new After(TakeNetworkChaosInformationTest.class, method),
+                reporterConfiguration);
+
+        //then
+        String actual = readJSONFileAsString(new File(path));
+        String expected = "{\"services\":[{\"actionon\":\"helloworld\",\"type\":\"Timeout\",\"phase\":\"create\",\"toxic\":{\"name\":\"timeout_downstream\",\"stream\":\"DOWNSTREAM\",\"toxcicity\":1.0,\"timeout\":1000}}]}";
+        JSONAssert.assertEquals(expected, actual, false);
```

Comment:
I think this we can make even better by using JsonPath assertions instead. Have a look here:
https://github.com/jayway/JsonPath/tree/master/json-path-assert#usage-guide